### DISCUSSION TITLE
[Feature Request] Add mouse wheel to standard layout

### DISCUSSION THREAD

**Discussion by a8ksh4** [manna-harbour/miryoku#64] [Q&A] [answered]
Not sure if I'm missing this in the current layout already...  It would be nice to add key mappings for mouse wheel up/down to the layout.  There are lots of web elements that are only easily interactable by scrolling the mouse wheel.

Could add this to the m and k keys (left most column for right hand, middle and bottom keys).

Thanks!

**Comment by manna-harbour** [comment, marked as answer]
The mouse wheel keys are on the bottom row of the [Mouse layer](https://github.com/manna-harbour/miryoku/tree/master/docs/reference#mouse).

**Reply by a8ksh4** [reply]
Thanks!
